core/services/feeds/models.go

```diff
@@ -118,6 +118,16 @@ type FeedsManager struct {
 	IsConnectionActive bool
 	CreatedAt          time.Time
 	UpdatedAt          time.Time
+	IsEnabled          bool
+}
+
+// Partial Feeds Manager is used to update Feeds Manager in the database.
+type PartialFeedsManager struct {
```

Comment:
not sure if it is worth embedding this partial into the object above , but in case u are curious https://gobyexample.com/struct-embedding, probably not worth doing as it changes the way `FeedsManager` get set?

Reply:
The field "isEnabled" is a bool pointer here and a bool in FeedsManager.
I think the "correct" way would to have different structs for different layers (services, graphql and repo) but that would increase the PR scope.

Reply:
make sense!